performance/load_test_performance_analysis.py

```diff
@@ -0,0 +1,365 @@
+    columns = cli.get_machine_schema(machines[0])["display"][:].to_list()
+
+    # Define query parameters
+    query = {
+        "Machine": machines[0],
+        "End Time__gte": datetime(2023, 4, 1),
+        "End Time__lte": datetime(2023, 4, 2),
+        "_order_by": "-End Time",
+        "_limit": 100,
+        "_only": columns,
+    }
+
+    return perform_test(get_cycles, config, cli, **query)
+
+
+def perform_get_line_data_load_test(
+    config: t_.Dict[str, t_.Any], cli: Client
+) -> t_.Dict[str, t_.Any]:
+    START_DATETIME = "2023-04-01T08:00:00.000Z"
+    END_DATETIME = "2023-04-02T23:00:00.000Z"
+    TIME_ZONE = "America/Los_Angeles"
+    MACHINE4 = "JB_NG_PickAndPlace_1_Stage4"
+    FIELD_NAME1 = "stats__BLOCKED__val"
+    FIELD_NAME2 = "stats__PneumaticPressure__val"
+    MIN_PRESSURE = 75.25
+    MAX_ROWS = 50
+
+    assets = [MACHINE4]
+    fields = [
+        {"asset": MACHINE4, "name": FIELD_NAME1},
+        {"asset": MACHINE4, "name": FIELD_NAME2},
+    ]
+
+    time_selection = {
+        "time_type": "absolute",
+        "start_time": START_DATETIME,
+        "end_time": END_DATETIME,
+        "time_zone": TIME_ZONE,
+    }
+
+    filters = [
+        {
+            "asset": MACHINE4,
+            "name": FIELD_NAME2,
+            "op": "gte",
+            "value": MIN_PRESSURE,
+        }
+    ]
+
+    query = {
+        "assets": assets,
+        "fields": fields,
+        "time_selection": time_selection,
+        "filters": filters,
+        "limit": MAX_ROWS,
+    }
+
+    return perform_test(get_line_data, config, cli, **query)
+
+
+def perform_get_kpi_data_viz_load_test(
+    config: t_.Dict[str, t_.Any], cli: Client
+) -> t_.Dict[str, t_.Any]:
+    machine_sources = ["Nagoya - Pick and Place 6"]
+    kpis = ["quality"]
+    i_vars = [
+        {
+            "name": "endtime",
+            "time_resolution": "day",
+            "query_tz": "America/Los_Angeles",
+            "output_tz": "America/Los_Angeles",
+            "bin_strategy": "user_defined2",
+            "bin_count": 50,
+        }
+    ]
+    time_selection = {
+        "time_type": "relative",
+        "relative_start": 7,
+        "relative_unit": "year",
+        "ctime_tz": "America/Los_Angeles",
+    }
+
+    data_viz_query = {
+        "where": [],
+        "db_mode": "sql",
+    }
+
+    query = {
+        "machine_sources": machine_sources,
+        "kpis": kpis,
+        "i_vars": i_vars,
+        "time_selection": time_selection,
+        "where": [],
+        "db_mode": "sql",
+    }
+
+    return perform_test(get_kpi_data_viz, config, cli, **query)
+
+
+# Main function
+def main(config_file: t_.Optional[str], xml_file: t_.Optional[str]) -> None:
+    # Read configurations
+    config = read_config(config_file)
+
+    # Get environment variables
+    tenant = os.environ.get("ENV_VAR_TENANT", "")
+    api_key = os.environ.get("ENV_VAR_API_KEY", "")
+    api_secret = os.environ.get("ENV_VAR_API_SECRET", "")
```

Comment:
See comment on readme but these envs should prefix with `ENV_SDK` instead of `ENV`

Reply:
Done.